lib/rules/jsx-no-target-blank.js

```diff
@@ -33,8 +33,21 @@ module.exports = {
           var relFound = false;
           var attrs = node.parent.attributes;
           for (var idx in attrs) {
-            if (attrs[idx].name && attrs[idx].name.name === 'rel') {
-              var tags = attrs[idx].value.type === 'Literal' && attrs[idx].value.value.toLowerCase().split(' ');
+            if (!Object.prototype.hasOwnProperty.call(attrs, idx)) {
```

Comment:
In that case I'd prefer to rewrite it to use `.filter` in lieu of the continues, and `.find` in lieu of the breaks and returns.

However, to keep this PR small, we can stick with this guard (which should have been here in the first place).